workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -379,7 +402,20 @@ def _compute_postphysics(self) -> Diagnostics:
             (self._tendencies, diagnostics, state_updates,) = self._postphysics_stepper(
                 self._state.time, self._state
             )
+            state_updates = _replace_precip_rate_with_accumlation(
+                state_updates, self._timestep
+            )
+
+            self._log_info(
```

Comment:
Maybe use debug level logging for this information to avoid adding even more to the INFO level logs.